safe-global/safe-wallet-web
# Feat: blink title if walletconnect needs attention

<img width="653" alt="Screenshot 2023-05-24 at 10 17 25" src="https://github.com/safe-global/safe-wallet-web/assets/381895/bc5eb755-f02e-41f6-af24-e9056ec8fee4">

If a SafeAppReviewModal or a MsgModal appear in a hidden tab, the title will start blinking so that the user knows they need to switch to that tab and confirm the tx/message signature.

## How to test

* Open https://blink--walletweb.review-wallet-web.5afe.dev/
* Connect your wallet and open a Safe account
* Go to the WalletConnect safe app
* Open Snapshot.org in another tab
* Connect to Snapshot via the WalletConnect safe app
* Edit your Snapshot profile name
* Observe that the Safe tab starts blinking

<video src="https://github.com/safe-global/safe-wallet-web/assets/381895/94d599bd-c795-4b16-b450-5647c24896e9" />
